src/bootstrap/src/core/build_steps/doc.rs

```diff
@@ -1024,10 +1024,6 @@ macro_rules! tool_doc {
                         run.builder.ensure(Rustc::from_build_compiler(run.builder, compilers.build_compiler(), target));
                         compilers.build_compiler()
                     }
-                    Mode::ToolBootstrap => {
-                        // bootstrap/host tools should be documented with the stage 0 compiler
```

Comment:
Maybe have a comment somewhere emphasizing that everything needs to be documented with the same rustdoc?